packages/azure-http-specs/spec-summary.md

```diff
@@ -800,6 +800,60 @@ maxpagesize=3
 }
 ```
 
+### Azure_ResourceManager_CommonProperties_Error_createOrReplace
+
+- Endpoint: `put https://management.azure.com`
+
+Resource PUT operation.
+Expected path: /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/test-rg/providers/Azure.ResourceManager.CommonProperties/confidentialResources/confidential",
+Expected query parameter: api-version=2023-12-01-preview
+Expected request body:
+
+```json
+{
+  "location": <any string>,
+  "properties": {
+    "username": "my_username",
+    "password": "my_password"
+  }
+}
+```
+
+Expected response status code: 403
+Expected response body:
+
+```json
+{
+  "error": {
+    "code": "AuthorizationFailed",
+    "message": "Your account or service principal doesn't have sufficient access to complete the operation.",
+    "innerError": {
+      "exceptiontype": "general"
+    }
```

Comment:
this part seems not ARM?
https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v6/types.json#L83-L135

Reply:
No, `CloudError` is to test custom defined error. 
e.g. Compute: https://github.com/Azure/azure-rest-api-specs/pull/32748/files#diff-bd60c6106ada2ea78b87a4cf98040f192149df43b988f22ec3c34c7a50729bf6R29

Normal ARM exception is tested in `GET` operation.

Reply:
Sure. 400 should be enough.